conf/modules/sonar_vl53l1x.xml

```diff
@@ -27,7 +27,8 @@
   </header>
 
   <init fun="sonar_vl53l1x_init()"/>
-  <periodic fun="sonar_vl53l1x_read()" period="0.020"/>
+  <periodic fun="sonar_vl53l1x_read()" freq="10."/>
```

Comment:
Could you still revert this periodic frequency back to 50Hz? I think it would be good to leave the option open for higher update rates. For instance, the trashcan I'm working on is very quick and probably also has a lot of vibrations on the accelerometer, so it might benefit from a higher AGL update rate (not tested, though). The default intermeasurement timing of 100ms will limit the update rate to 10Hz anyway in normal cases, so changing the periodic frequency would not change anything during normal use, but would give room for higher update rates when required.

Reply:
I changed it back to 50Hz, it seems fine even with the multi_ranger deck.
As a side note, I should really make these parameters configurable from airframe file as well...

Also @tomvand , do you know if someone is already working on the optic flow sensor driver (pmw3901) ?

Reply:
Thanks. Yes that would be nice for some cases.

I started work on the pmw3901 yesterday ([here](https://github.com/tomvand/paparazzi/tree/pmw3901mb)). No output yet, not sure if it's a software or wiring issue (had to solder far too tiny wires). With a bit of luck I hope to finish the driver some time next week.